### DISCUSSION TITLE
using tailwind with vue

### DISCUSSION THREAD

**Discussion by Nethrenial** [alex8088/electron-vite#201] [Q&A]
Using tailwind with vue doesn't trigger style changes with HMR, but when stopping the dev server and starting again, it shows the correct style. Do you know how I can fix it?

**Comment by alex8088** [comment]
Project Structure ? Is css work ?

**Comment by roma-marshall** [comment]
@Nethrenial try this in your project folder: `npm run dev` or `yarn run dev`
